tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Neal Chambers
```

Comment:
Could you rename this file to something more descriptive?

Reply:
I'm pretty sure that is what it has to be named in order to run before all tests are run. There isn't a lot of documentation on it, but that is what [this comment](https://stackoverflow.com/questions/34466027/what-is-conftest-py-for-in-pytest#:~:text=Yeah%2C%20the%20documentation,at%2014%3A46) suggests. Pytest documentation just has this [snippet](https://docs.pytest.org/en/7.1.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files) on it for fixtures, but pytest-recording suggests this file as where to put the VCR config.